exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java

```diff
@@ -61,6 +60,7 @@
 import org.junit.runners.Parameterized;
 
 @RunWith(Parameterized.class)
+
```

Comment:
new line?

Reply:
Actually not required, tried to add another RunWith for Mocking and removed later on leaving the newline.

Reply:
ok, just remove it